homeassistant/components/matter/climate.py

```diff
@@ -76,6 +76,8 @@ class MatterClimate(MatterEntity, ClimateEntity):
     _attr_supported_features: ClimateEntityFeature = (
         ClimateEntityFeature.TARGET_TEMPERATURE
         | ClimateEntityFeature.TARGET_TEMPERATURE_RANGE
+        | ClimateEntityFeature.TURN_OFF
+        | ClimateEntityFeature.TURN_ON
```

Comment:
Adding the off feature makes sense, as it is hardcoded at line 94, but the on feature depends a bit, looking below, it doesn't have to be available?

Reply:
Yes, I'd only support the turn on feature if we add any other hvac mode than off.